test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -786,7 +788,7 @@ public void Editor_InputTypeDateTime_RendersAsDateTime()
                 hour: 3,
                 minute: 4,
                 second: 5,
-                millisecond: 6,
+                millisecond: 60,
```

Comment:
FYI I made a few of these changes to make the `DateTime` and `DateTimeOffset` values consistent.